speeduino/init.ino

```diff
@@ -1267,6 +1267,7 @@ void setPinMapping(byte boardID)
       pinTrigger3 = 3; //The Cam sensor 2 pin
       pinTPS = A2;//TPS input pin
       pinMAP = A3; //MAP sensor pin
+      pinBaro = A5; //Baro Sensor Input
```

Comment:
You are adding pinBaro (A5) twice, here (Line 1270) and again later on (Line 1291).